lib/active_triples/rdf_source.rb

```diff
@@ -39,21 +41,17 @@ module RDFSource
     extend ActiveSupport::Concern
 
     include NestedAttributes
+    include Persistable
     include Properties
     include Reflection
+    # include RDF::Term
```

Comment:
Sort of, there's a parallel branch at:
https://github.com/ActiveTriples/ActiveTriples/compare/feature/rdf-term-compatibility#diff-e9939ce86c000df5686160af00e0c53fR221

Some of the work to bring the RDF.rb model interfaces in line had to happen to fix some problems that arose in this PR, so things are a little bled together.

It would probably be good to rebase the whole thing significantly before merge.